### DISCUSSION TITLE
After AFL stops, how to continue testing based on the previous queue?

### DISCUSSION THREAD

**Discussion by blu3sh0rk** [AFLplusplus/AFLplusplus#1788] [Q&A]
After AFL stops, how to continue testing based on the previous queue? Besides using the queue as new input, are there any other mechanisms or methods?

And how to add new seeds to the queue and continue testing?

**Comment by domenukk** [comment]
Use the same `-o` out folder and node name and then either set the env variable `AFL_AUTORESUME=1` or use `-i -` as input path
